types/dynamodb/Model.d.ts

```diff
@@ -0,0 +1,147 @@
+import { AnySchema } from 'joi';
+import * as bunyan from 'bunyan';
+import { GetItemInput, DynamoDB } from './DynamoDB';
+
+import { Callback } from './Callback';
+import { Query } from './Query';
+import { Scan, ParallelScan } from './Scan';
+import { EventEmitter } from 'events';
+
+export class Item extends EventEmitter {
+    constructor(attrs: object, table: any);
+    get(name: string): any;
+    set(params: any): this;
+    save(callback: Callback): void;
+    save(): Promise<any>;
+    update(options: any, callback: Callback): void;
+    update(options: any): Promise<any>;
+    destroy(options: any, callback: Callback): void;
+    destroy(options: any): Promise<any>;
+    toJSON(): object;
+}
+
+export class Model extends Item {
+    constructor(attrs: any);
+}
+
+export namespace Model {
+    const get: GetOperation;
```

Comment:
Aren't these consts and functions equivalent to statics on `class Model`? This is perfectly legal but it's an odd style.

Typescript doesn't support nested types in classes, so the types do need to be in a namespace.